cmake/HPX_SetupTarget.cmake

```diff
@@ -70,12 +70,12 @@ function(hpx_setup_target target)
   get_target_property(target_SOURCES ${target} SOURCES)
 
   # Manage files with .cu extension in case When Cuda Clang is used
-  if(target_SOURCES AND HPX_WITH_CUDA_CLANG)
+  if(target_SOURCES AND HPX_WITH_CLANG_CUDA)
     foreach(source ${target_SOURCES})
       get_filename_component(extension ${source} EXT)
       if(${extension} STREQUAL ".cu")
         set_source_files_properties(
-          ${source} PROPERTIES COMPILE_FLAGS "${HPX_CUDA_CLANG_FLAGS}"
+          ${source} PROPERTIES COMPILE_FLAGS "${CMAKE_CUDA_FLAGS}"
```

Comment:
Isn't the point of `CMAKE_CUDA_FLAGS` that it gets applied automatically to all files that are of type `CUDA`?